src/libs/ajax/ajax-common.ts

```diff
@@ -191,4 +191,6 @@ export const fetchEcm = _.flow(
 
 export const fetchGoogleForms = withUrlPrefix('https://docs.google.com/forms/u/0/d/e/', fetchOk)
 
-export const fetchWDS = _.flow(withUrlPrefix(`${getConfig().wdsUrlRoot}/`), withAppIdentifier)(fetchOk)
+export const fetchWDS = wdsProxyUrlRoot => _.flow(withUrlPrefix(`${wdsProxyUrlRoot}/`),
+  withRetryAfterReloadingExpiredAuthToken,
+)(fetchOk)
```

Comment:
Currently, calls to `fetchWDS` look like:
```ts
getSchema: async (root: string, instanceId: string): Promise<RecordTypeSchema[]> => {
    const res = await fetchWDS(root)(`${instanceId}/types/v0.2`, _.merge(authOpts(), { signal }))
```

Where `fetchWDS(root)` returns a function that is called in the same expression. Since the caller is passing in both the `root` URL prefix and the relative path, would it make sense to get rid of there `withUrlPrefix` here?

```suggestion
export const fetchWDS = withRetryAfterReloadingExpiredAuthToken(fetchOk)
```

So that the call becomes less convoluted:
```ts
getSchema: async (root: string, instanceId: string): Promise<RecordTypeSchema[]> => {
    const res = await fetchWDS(`${root}/${instanceId}/types/v0.2`, _.merge(authOpts(), { signal }))
```